### PR TITLE
Update Crons to include platform docs

### PR DESCRIPTION
**Pull request by gaprl**
Adds the following dedicated platform docs for Crons:
- PHP
- Python
- Celery Beat

Also adds two new pages for setting up Crons with HTTP and CLI.

### CODE REVIEW

File: src/platform-includes/crons/setup/php.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+$checkIn = new CheckIn(
+    monitorSlug: '<monitor-slug>',
+    status: CheckInStatus::ok(),
+);
+$event->setCheckIn($checkIn);
+$this->hub->captureEvent($event);
+
+```
+
+If your job execution fails, you can do:
```
Comment (lizokm) [inline src/platform-includes/crons/setup/php.mdx:60]:
```suggestion
If your job execution fails, you can:
```

File: src/platform-includes/crons/setup/python.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## Job Monitoring
+
+Use the Python SDK to monitor and notify you if your job doesn't start when expected (missed), fails, or if it exceeded its maximum runtime (failed).
+
```
Comment (lizokm) [inline src/platform-includes/crons/setup/python.mdx:4]:
```suggestion
Use the Python SDK to monitor and notify you if your job is missed (or doesn't start when expected), if it fails due to a problem in the runtime (such as an error), or if it fails by exceeding its maximum runtime.

```

File: src/platform-includes/crons/setup/php.laravel.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+## Job Monitoring
+
+Use the Laravel integration to monitor and notify you if your job doesn't start when expected (missed), fails, or if it exceeded its maximum runtime (failed).
```
[the remark this answers is not in the document]
Comment (gaprl) [reply src/platform-includes/crons/setup/php.laravel.mdx:3]: will merge liza's suggestion with the link provided

File: src/platform-includes/crons/setup/php.mdx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+$event = Event::createCheckIn();
+$event->setCheckIn(new CheckIn(
+    id: $checkIn->getId(),
+    monitorSlug: '<monitor-slug>',
+    status: CheckInStatus::ok(),
+));
+$this->hub->captureEvent($event);
+```
+
+If your job execution fails, you can do:
```
Comment (lizokm) [inline src/platform-includes/crons/setup/php.mdx:27]:
```suggestion
If your job execution fails, you can:
```

File: src/platforms/common/crons/index.mdx [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+
+`schedule`:
+
+: The job's schedule based on the `schedule_type`:
+- `crontab`: a string crontab expression with five fields, such as ` "0 * * * *"`.
+- `interval`: a tuple for your interval such as `[2, "hour"]`.
+
+`checkin_margin`:
+
+: Time (in minutes) of how long Sentry should wait for your check-in until it is considered missed ("grace period"). Optional.
```
Comment (lizokm) [inline src/platforms/common/crons/index.mdx:157]:
```suggestion
: The amount of time (in minutes) Sentry should wait for your check-in before it's considered missed ("grace period"). Optional.
```

File: src/platforms/common/crons/index.mdx [hunk 6]
```diff
@@ -0,0 +1,332 @@
...
+
+<Note>
+
+We recommend that your check-in margin to be less or equal to your interval.
```
Comment (lizokm) [inline src/platforms/common/crons/index.mdx:161]:
```suggestion
We recommend that your check-in margin be less than or equal to your interval.
```

File: src/platforms/common/crons/index.mdx [hunk 7]
```diff
@@ -0,0 +1,332 @@
...
+
+</Note>
+
+`max_runtime`:
+
+: Time (in minutes) your job is allowed to run before it is considered failed. Optional.
```
Comment (lizokm) [inline src/platforms/common/crons/index.mdx:167]:
```suggestion
: The amount of time (in minutes) your job is allowed to run before it's considered failed. Optional.
```

File: src/platforms/common/crons/index.mdx [hunk 8]
```diff
@@ -0,0 +1,332 @@
...
+
+`timezone`
+
+: The `tz` where your job is running. This is usually your server's timezone, such as `America/Los_Angeles`. See [list of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). Optional.
```
Comment (lizokm) [inline src/platforms/common/crons/index.mdx:171]:
```suggestion
: The `tz` where your job is running. This is usually your server's timezone, (such as `America/Los_Angeles`). See [list of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). Optional.
```

File: src/platforms/common/crons/index.mdx [hunk 9]
```diff
@@ -0,0 +1,332 @@
...
+Content-Type: application/json
+
+{
+    "status": "failed"
+}
+```
+
+## Connecting Errors to Cron Monitors
+
+If you're capturing Sentry errors, you can link them to a monitor by using context.
```
Comment (lizokm) [inline src/platforms/common/crons/index.mdx:292]:
```suggestion
If you're capturing Sentry errors, you can link them to a Cron Monitor by using context.
```

**Review by lizokm — APPROVED** [review]
We're SUPER close!

**Pull request merged**